test/lib/jdk/test/lib/security/CertificateBuilder.java

```diff
@@ -439,13 +459,12 @@ private byte[] encodeTopLevel(X509Certificate issuerCert,
             PrivateKey issuerKey, String algName)
             throws CertificateException, IOException, NoSuchAlgorithmException {
 
-        AlgorithmId signAlg = AlgorithmId.get(algName);
+        AlgorithmId signAlg;
```

Comment:
This variable looks like it is unused now.

Reply:
It's used on 476 for encoding the AlgorithmIdentifier of the certificate signature, and also on 530 to again provide that same AlgorithmIdentifier encoding in the TBSCertificate strructure.

Reply:
You mean lines 458 and 466? Yes, I missed that.